hack/sdk_setup.sh

```diff
@@ -9,32 +11,50 @@ pushd /tmp
 GOPATH=${GOPATH:-/root/golang}
 export GOPATH=$GOPATH
 
-mkdir -p $GOPATH/bin
-mkdir -p $GOPATH/src/github.com/openshift
-mkdir -p $GOPATH/src/github.com/operator-framework
-
-sudo yum install -y git
-sudo curl https://dl.google.com/go/go1.10.3.linux-amd64.tar.gz -o go1.10.3.linux-amd64.tar.gz
-sudo tar -C /usr/local -xzf go1.10.3.linux-amd64.tar.gz
-
-export PATH=$PATH:/usr/local/go/bin:$GOPATH/bin
-sudo curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-popd
-
-pushd $GOPATH/src/github.com/operator-framework
-git clone https://github.com/operator-framework/operator-sdk
-popd
-
-pushd $GOPATH/src/github.com/operator-framework/operator-sdk
-git checkout master
-make dep
-make install
+if [ ! -d $GOPATH/bin ] ; then
+    mkdir -p $GOPATH/bin
+fi
+
+type -p git || sudo yum install -y git
+
+needgo=1
+case "$( go version 2>&1 )" in
+*go1.10.*) needgo= ;;
+esac
+if [ -n "$needgo" ] ; then
+    sudo curl https://dl.google.com/go/go1.10.5.linux-amd64.tar.gz -o go1.10.5.linux-amd64.tar.gz
+    sudo tar -C /usr/local -xzf go1.10.5.linux-amd64.tar.gz
+    export PATH=/usr/local/go/bin:$GOPATH/bin:$PATH
+fi
+type -p dep || { sudo curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh ; }
 popd
 
-pushd $GOPATH/src/github.com/openshift
-git clone https://github.com/$ESO_OPERATOR_REPO/elasticsearch-operator --branch $ESO_OPERATOR_BRANCH
-git clone https://github.com/$CLO_OPERATOR_REPO/cluster-logging-operator --branch $CLO_OPERATOR_BRANCH
-popd
+if [ ! -d $GOPATH/src/github.com/openshift/elasticsearch-operator ] ; then
+  git clone https://github.com/$ESO_OPERATOR_REPO/elasticsearch-operator \
+    --branch $ESO_OPERATOR_BRANCH \
+    $GOPATH/src/github.com/openshift/elasticsearch-operator
```

Comment:
this will create the missing paths if necessary? :+1:

Reply:
yes